react-app-rewired/headers/csps/nameServices/yat.ts

```diff
@@ -0,0 +1,5 @@
+import type { Csp } from '../../types'
+
+export const csp: Csp = {
+  'connect-src': ['https://octopus-app-mkjlj.ondigitalocean.app'],
```

Comment:
this needs to point to something more stable that is maintained by yat and not temporary, we will not merge this as is.

Reply:
Alrighty, who can discuss with them to get something stable from YAT and to get whitelisted from YAT side. First to avoid cors error, secondly to avoid the security error when trying to call YAT api

Reply:
@saksijas we're handling this on the vendor side 🤘